frontend/src/v5/store/tickets/card/ticketsCard.redux.ts

```diff
@@ -93,6 +94,13 @@ export const setPinToDrop = (state: ITicketsCardState, { pinToDrop }: SetPinToDr
 };
 
 const getFilterKey = ({ module, property, type }: CardFilter): TicketFilterKey => `${module}.${property}.${type}`;
+export const setFilters = (state: ITicketsCardState, { filters }: SetFiltersAction) => {
+	filters.forEach((filter) => {
+		const path = getFilterKey(filter);
+		state.filters[path] = filter.filter;
+	});
+};
+
```

Comment:
I dont like having a redux action for something that you could already do with other actions (upsert filter). you could just do  `filters.foreach(TicketsCardActionsDispatchers.upsertFilter)` in defaultticketfiltersSetter  instead of adding a specific action for setting all the filters (unless we come up with a different use for TicketsCardActionsDispatchers.setFilters).